### DISCUSSION TITLE
Books marked missing, even though they are actually present!

### DISCUSSION THREAD

**Discussion by maconly34** [advplyr/audiobookshelf#3503] [Q&A] [answered]
How do I fix the database showing 39 missing books, even though they are present on the server.
I have tried scanning and searching for books, but they are still "missing".
Thanks for your help

**Comment by nichwall** [comment, marked as answer]
Are you rescanning the entire library or the individual book? There is a "rescan" button on the edit popup for books.

If the library scan does not find it but the individual book scan finds it, the files are probably included under another book. This usually happens due to not following the structure of 1 book per folder, either initially or when adding a new book later, causing books in subfolders from that book to be combined.

You can delete the "problem book" which includes the missing books, fix the file structure, and then rescan.

**Reply by maconly34** [reply]
Many thank's for this quick answer solution!
Work with individual re-scan!
MERCI
